### ISSUE TITLE
demo not working

### ISSUE THREAD

**Issue by materialvision** [Fenny/ChromecastJS#1]
Hi, I can't seem to get the demo working. I get following error in console: Uncaught TypeError: Cannot read property 'CastContext' of undefined
Any ideas?

**Comment by Fenny** [comment]
Hey,

Are you trying it in chrome?

**Comment by materialvision** [comment]
Yes, but I restarted and tried again since I already had a lot of casts up and running in other tabs. It works now, sorry should have tested better!

**Comment by Fenny** [comment]
@materialvision I updated the library, found a small bug.
Should work now, could you test it?

https://fenny.github.io/ChromecastJS/demo/index.html
